index.html

```diff
@@ -16,26 +16,32 @@
     <script type="module" src="arrayList.js"></script>
```

Comment:
You deleted `arrayList.js` so you should remove the corresponding `script` tag. You can see that it throws an error in the console:
<img width="568" alt="image" src="https://github.com/user-attachments/assets/900f4ab6-add1-46c3-a5ea-84a6270f355d">

Reply:
I Am so sorry . I totally forgot to delete it.